### PR TITLE
doc: make sure configmap object referred or documented correctly

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal.

File: examples/rbd/storageclass.yaml [hunk 1]
```diff
@@ -42,21 +42,17 @@ parameters:
...
    # By default it is disabled. Valid values are "true" or "false".
    # A string is expected here, i.e. "true", not true.
    # encrypted: "true"
-
    # Use external key management system for encryption passphrases by specifying
    # a unique ID matching KMS ConfigMap. The ID is only used for correlation to
-   # config map entry.
+   # configmap entry.
    # encryptionKMSID: <kms-config-id>
-
```
Comment (Madhu-1) [inline examples/rbd/storageclass.yaml:59]: why new lines are removed in both storageclass?

File: docs/deploy-rbd.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ make image-cephcsi
 **Available command line arguments:**
 
 | Option                | Default value         | Description                                                                                                                                                                                                                                                                          |
-|-----------------------|-----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --------------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (Madhu-1) [inline docs/deploy-rbd.md:30]: not sure why it was not caught in CI. @humblec is it possible to update mdlinter to catch this one?

File: examples/rbd/storageclass.yaml (hunk 1 above)
Comment (humblec) [reply examples/rbd/storageclass.yaml:59]: We could keep as it is too.. Let me revert this change.

File: docs/deploy-rbd.md (hunk 2 above)
Comment (humblec) [reply docs/deploy-rbd.md:30]: I am not sure why it was caught on CI. may be a version difference of mdlinter.

File: examples/rbd/storageclass.yaml (hunk 1 above)
Comment (humblec) [reply examples/rbd/storageclass.yaml:59]: @Madhu-1 done.

File: docs/deploy-rbd.md (hunk 2 above)
Comment (Madhu-1) [reply docs/deploy-rbd.md:30]: maybe, can you please check why it was not caught. if it's a mdlinter issue can you please update mdlinter to the same version where you are hitting this issue.

**Comment by humblec** [comment]
Closing this PR for avoiding wrong push.

**Pull request closed without merging**